js/iframe-coordinator/Elm.d.ts

```diff
@@ -1,7 +1,61 @@
 declare module "*/Client.elm" {
-  export var Client: any;
+  export interface LabeledMsg {
```

Comment:
Moved these interfaces into the modules since thats where they are really coming from.